plugins/bundle/status.go

```diff
@@ -30,6 +33,7 @@ type Status struct {
 	Message                  string          `json:"message,omitempty"`
 	Errors                   []error         `json:"errors,omitempty"`
 	Metrics                  metrics.Metrics `json:"metrics,omitempty"`
+	HTTPCode                 json.Number     `json:"http_code,omitempty"`
```

Comment:
❓ Why `json.Number` and not `int`?

Reply:
I used `json.Number` in combination with `omitempty` to omit marshalling the field if `HTTPCode` is not explicitly set.
If I used `int`, `HTTPCode` would be marshalled to the default value, 0:
```
"bundles": {
      "myBundle": {
        "name": "myBundle",
        "last_successful_activation": "0001-01-01T00:00:00Z",
        "last_successful_download": "0001-01-01T00:00:00Z",
        "last_successful_request": "0001-01-01T00:00:00Z",
        "last_request": "2022-01-20T16:10:22.093047Z",
        "code": "bundle_error",
        "http_code": 0,
 ```

Reply:
💡 So with `json.Number` we'll get the actual number, not a string...? I guess I've just gotten confused in the tests.

Reply:
Yes, it's a bit confusing at first, since `json.Number` is a `string` internally. However, it will be marshalled as a number, e.g.
```
"bundles": {
      "myBundle": {
        "name": "myBundle",
        "last_successful_activation": "0001-01-01T00:00:00Z",
        "last_successful_download": "0001-01-01T00:00:00Z",
        "last_successful_request": "0001-01-01T00:00:00Z",
        "last_request": "2022-01-20T16:10:22.093047Z",
        "code": "bundle_error",
        "http_code": 403,
```

Reply:
Perfect, thanks for bearing with me.